build/tasks/validate.js

```diff
@@ -27,6 +27,27 @@ function hasUniqueId() {
 	};
 }
 
+function hasTwoOutcomes(messages) {
```

Comment:
Looks like this is normalising multiple outcomes, should change function name to something generic like `normaliseOutcomes` or `hasMultipleOutcomes` rather than exactly `Two`?